kolibri/core/assets/src/core-app/apiSpec.js

```diff
@@ -90,6 +90,9 @@ module.exports = {
         iconButton: {
           module: require('../vue/icon-button'),
         },
+        textbox: {
```

Comment:
Is it text box or textbox? To camel or not to camel... I am inclined to the former.